### ISSUE TITLE
WCAG 2.0 contrast failures

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#121]
WebAIM's WAVE (web accessibility checker) has identified several contrast issues on the UW home page. See the [WAVE Report](http://wave.webaim.org/report#/uw.edu) for details. WAVE adds icons to the page to flag problematic content. In the left sidebar, there's a tool for navigating through the issues. In the sidebar, click the "Contrast" button to see the various contrast issues. There are essentially two general problems: 

1. Content where contrast is legitimately too low (e.g. black on brown "Quick Links"; brown on tan text in the News & Events boxes). 
2. Content where text appears over a background image. In this case, the content block should have a defined background-color that has WCAG-compliant contrast with the foreground text. The background color is important because it serves as a fallback in case the background image fails to load.

**Comment by lcaple** [comment]
I was able to fix most of these contrast issues on the homepage's child theme. 

Keeping this open as the Quicklinks contrast issue is still outstanding.

**Comment by lcaple** [comment]
This was addressed in out last push for accessibility updates on the homepage & in uw-2014.

**Comment by terrill** [comment]
I'm seeing new contrast issues in the new theme, as measured by the WAVE browser extension on both the [Admissions](https://www.washington.edu/admissions/) and [CISO](https://ciso.uw.edu/) websites. The Admissions site has 5 errors, whereas the CISO site has 11. The 5 errors on the Admissions site are shared on both sites. Keep in mind that some of these errors may be the result of a CSS background-color either having too little contrast or not being defined at all, even if the background-color is hidden by a background image. This is problematic when the image fails to load, or is slow to load, and users need to be able to read the text.  f

**Comment by lcaple** [comment]
Hi Terill,

The Admissions site uses a different child theme that's heavily customized and based off of uw-2014. I will forward these to Tiffany Sevareid who manages that site and let her know.

The CISO site is actually using the new UW WordPress Theme which you can find here: https://github.com/uweb/uw_wp_theme

The codebases are different enough that it's better for us to track updates for that theme over there in the future! I'll copy a note about this over to the issue queue there.

**Comment by terrill** [comment]
I'm seeing a few on the Accessible Technology site too, still under development.  And on this page: 
http://cmsdev.uw.edu/cms/newtheme-ats2/new-shortcodes/

On the latter, WAVE finds 7 contrast errors. Here are the offending elements: 

- a.uw-patch
- a.uw-workmark
- button.uw-quicklinks
- div.uw-site-title
- label.screen-reader-text (The "search for" label; this one of course is intended not to be seen so maybe it doesn't apply) 
- a.footer-wordmark
- h3.be-boundless
